pi_local.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# This script will log in the address of a raspberry-pi 400 ubuntu server, copy the files over the Pi, excute the script, and SSH into the server.
+PI_HOST=pi
+echo -e "\n==== Validate Pi server is running ====\n"
+while true
+do
+  if ( ping -c 1 $PI_HOST > /dev/null 2>&1 )
```

Comment:
Can you switch this to `ssh user@host 'exit'` to use ssh to see if you can connect? There is also a check like this for example:

https://docs.github.com/en/authentication/connecting-to-github-with-ssh/testing-your-ssh-connection

Reply:
You've successfully authenticated, but GitHub does not provide shell access.

Reply:
ssh: Could not resolve hostname host: nodename nor servname provided, or not known